app/views/layouts/application.haml

```diff
@@ -16,6 +16,12 @@
         - react_component_cache(name: "onboarding_topbar", props: props) do
           = react_component("OnboardingTopBar", props: props, prerender: true)
 
+  - if feature_enabled?(:topbar_v1)
+    - props = topbar_props
+    = javascript_include_tag "https://maps.googleapis.com/maps/api/js?libraries=places"
```

Comment:
This is a different URL from the one used in index.haml above. Which one should we use?